sdk/cosmos/azure-cosmos/samples/document_management_async.py

```diff
@@ -85,25 +85,52 @@ async def read_items(container):
 async def query_items(container, doc_id):
     print('\n1.4 Querying for an  Item by Id\n')
 
-    # enable_cross_partition_query should be set to True as the container is partitioned
+    # The enable_cross_partition_query option is not used by the async client - this is the default behavior
     # In this case, we do have to await the asynchronous iterator object since logic
     # within the query_items() method makes network calls to verify the partition key
     # definition in the container
     query_items_response = container.query_items(
         query="SELECT * FROM r WHERE r.id=@id",
         parameters=[
-            { "name":"@id", "value": doc_id }
-        ],
-        enable_cross_partition_query=True
+            {"name": "@id", "value": doc_id}
+        ]
     )
 
     items = [item async for item in query_items_response]
 
     print('Item queried by Id {0}'.format(items[0].get("id")))
 
 
+async def query_items_with_continuation_token(container):
+    print('\n1.5 Querying for Items using Pagination and Continuation Token\n')
+
+    # When using Pagination, max_item_count will limit the number of items in each page
+    query_iterable = container.query_items(
+        query="SELECT * FROM r",
+        max_item_count=1
+    )
+
+    item_pages = query_iterable.by_page()
+
+    # It is worth mentioning that from Python 3.10, you can use the syntax below to iterate as opposed to __anext__()
+    # first_page = await anext(item_pages)  # cspell:disable-line
+
+    first_page = await item_pages.__anext__()
```

Comment:
I samples, please use `anext(item_pages)`.
We can't use it in _our_ code, because we need to be backwards compatible - but that's not a requirement for samples.